File: 0001-two-sum/0001-two-sum.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize) 
{
    int key=0;
    int* index = (int*)malloc(2 * sizeof(int));
    for(int i =0;i<numsSize-1;i++)
    {
        key=0;
        for(int j=i+1;j<numsSize;j++)
        {   key=0;
            if(nums[i]+nums[j]==target)
            {
                index[0]=i;
                index[1]=j;
                *returnSize=2;
                return index;
            }
        }
    }
    *returnSize=0;
    return NULL;
}

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        
        """

        dictio = {}
    
        n = len(nums)
        for i in range(n):
            difference = target - nums[i] 
            if difference in dictio:
                return [dictio[difference], i]
            else:
                dictio[nums[i]] = i

        